app/views/projects/table.scala.html

```diff
@@ -73,33 +73,28 @@ <h3 class="panel-title">@messages("project.category.plural")</h3>
                         <a href="@routes.Application.index(None)"><i class="pull-right category-reset fa fa-times"></i></a>
                     }
                 </div>
-                <table class="table panel-body">
+                <table class="table panel-body category-table">
                     <tbody>
                         @for(i <- 0 until Categories.values.size) {
-                            <tr>
+                            @if(visibleCategories.isDefined) {
+                                @if(!visibleCategories.get.contains(Categories(i).asInstanceOf[Category])) {
+                                    <tr onclick="location.href='@routes.Application.index(Some(
```

Comment:
Yeah it is the only real way to get the entire row selected with out making multiple anchors.